linux_os/guide/system/accounts/accounts-banners/banner_etc_motd/tests/banner_etc_motd_disa_dod_default_banner.pass.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+#
```

Comment:
I suggest you remove these lines, you are removing them actually in tests for the other rule.

Reply:
This empty `#` in the header of test scripts is very common (at least 190 scripts have this now). I would suggest we document the standard for this: with or without this `#` when header parameters are not present so we can avoid doubts in the future. For now, I didn't find something. Are you aware of something? If they are there without any reason, I agree to remove what is not necessary, but not only here, everywhere.

Reply:
I agree we should settle on something. My point is that in this PR, you are removing these lines for one set of tests and adding them for another set of tests. I just want to keep it consistent in the PR :-)

Reply:
Ah, got your point. : ) Actually, I am not removing it in some files and including in others. I am actually incrementing the line with a parameter for some files which didn't have a parameter before. Inconsistent is prone to interpretation here, since we don't have a definition. hehe. In many cases, we only have a single "#" where no header parameter is necessary (I am following this since most of the scripts I checked follow this). However, checking all files we can also find many cases where this empty '#' is there together with other parameters. And, we have even a third case where nothing is there besides the shebang. Although this is nit pick, I think we should define a standard and gradually adjust everything according to the defined standard. I will suggest something for the Style Guide.

Reply:
https://github.com/ComplianceAsCode/content/pull/7639